testing/bfs/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: dai9ah <dai9ah@protonmail.com>
+# Maintainer: dai9ah <dai9ah@protonmail.com>
+pkgname=bfs
+pkgver=1.2.1
+pkgrel=0
+pkgdesc="Breadth-first variant of the UNIX find command"
+url="https://github.com/tavianator/bfs"
+arch="all"
+license="0BSD"
+makedepends="linux-headers"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/tavianator/$pkgname/archive/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	make -C "$builddir"
+}
+
+check() {
+	cd "$builddir"
+	./bfs -nohidden > /dev/null
```

Comment:
Why not `make -C "$builddir" check`?

Reply:
The contributor was probably too lazy and the committer didn’t check it during code-review. 😞
Could you please create a PR changing this to run `make check`?

Reply:
Done: https://github.com/alpinelinux/aports/pull/3596